libnavigation-core/src/main/java/com/mapbox/navigation/core/telemetry/LocationsCollectorImpl.kt

```diff
@@ -60,6 +60,7 @@ internal class LocationsCollectorImpl(
     }
 
     override fun onRawLocationChanged(rawLocation: Location) {
+        logger?.d(TAG, Message("onRawLocationChanged $rawLocation"))
```

Comment:
Do we need to keep this log? It will spam a lot.

Reply:
Agreed 👍

Reply:
Removed.